### DISCUSSION TITLE
Ballhead support

### DISCUSSION THREAD

**Discussion by StyXman** [Stellarium/stellarium#1905] [Q&A]
I'm always frustrated when I try to align the photos I take with my tripod, ballhead and DSLR. I know of the Ocular plugins, which with some work allows me to simulate my lens (as a telescope, calculating the diameter by hand as aperture/f_number), but I'm still limited to equatorial vs azimutal navigation, when the ballhead is completely freehand, no axis limitations.

What I would like to have is either a freehand 'browsing' mode where one can yaw (pan) left-right, pitch (tilt) up-down, roll (rotate) left-right and zoom in-out; or at least allow for the roll/rotate movement. 

To be fair, the only alternative to Stellarium I found (kstars) also has this same limitation.

**Comment by alex-w** [comment]
But CCD frame may rotate and you can use keyboard to move the frame in any direction. What is problem? Would you describe your use case?

**Comment by gzotti** [comment]
Free rotation of the view is not possible. Just configure a sensor and add a rotation.

**Comment by StyXman** [comment]
Wow, that was fast :)

Yes, I can rotate the frame, but I can't rotate the _view_. I can't even call myself an amateur astronomer/astrophotographer, but my head can't cope with arbitrary tilts when comparing two images. I just want to see in Stellarium as exactly as possible the same 'frame' that I see on my image browser.

**Comment by alex-w** [comment]
As said @gzotti free rotation of the view is not possible. I think no planetarium, which have this feature.

**Comment by gzotti** [comment]
Most of the time photos taken on a tripod with ballhead are in azimuthal mode. A slight image rotation can be easily configured as sensor rotation. There is then no other way for you than to make a screenshot with a rotated sensor frame and unrotate and crop this in Photoshop. This really a case of https://github.com/Stellarium/stellarium/wiki/FAQ#why-dont-you-implement

**Comment by StyXman** [comment]
> you can use keyboard to move the frame in any direction

how? I can't find any shortcut info in http://stellarium.sourceforge.net/wiki/index.php/Oculars_plugin .

**Comment by StyXman** [comment]
TBH, I was trying to move the _view_ so the frame would fall in the right place. Really frustrating.

**Comment by alex-w** [comment]
Please press F1 within the planetarium (or F7).
